### ISSUE TITLE
Enabling Watchdog for claymore miner in NHM 1.9.2.19

### ISSUE THREAD

**Issue by M1r00** [nicehash/NiceHashMiner#1908]
Hello,
I have a 6*GTX 1070s rig running W10 x64 1903. 
OC is GPU +0   MEM +600Mhz and PWR TGT 80%
I am using the latest NHM 1.9.2.19. 
I am experiencing issues with one of my GPU's  (GPU #1) that is hanging OPEN CL call. And I am tracking the issue to fix it. 
Meanwhile I tried to add arguments in the extra launch parameters for Claymore miner to enable Watchdog(-wd 1)  and auto restart ( -r >30)  , in order to restart Claymore in case the GPU 1 fails. I edited the MinerOptionsPackage.json to add these 2 options but it still fails to activate  watchdog or to auto restart the miner. Here is how edited the option package in the general_options: 

> {
>       "type": "OptionWithSingleParameter",   
>       "id": "claymoreDual_watchdog_option",
>       "short_name": "-wd",
>       "long_name": null,
>       "default_value": "1",
>       "delimiter": null
>     },
>     {
>       "type": "OptionWithSingleParameter",
>       "id": "claymoreDual_Restart_miner_mode",
>       "short_name": "-r",
>       "long_name": null,
>       "default_value": "0",
>       "delimiter": null
>     },

My logs shows that watchdog is disabled and miner doesn't restart when GPU fails.
Can someone show me a way to enable watchdog and autorestart for claymore in NHM? 
Thanks 

	M1r00  ;)

**Comment by Sfinx-vrn** [comment]
Yes, i have some problem too!!!!   
Screen: 
![2020-01-09_14-03-41](https://user-images.githubusercontent.com/24940598/72062525-045a0a80-32e9-11ea-9b4f-67be12308a84.png)

**Comment by luc1an24** [comment]
Hi @M1r00 
Thank you for reaching out.
Could you please send us logs of the time this error occurs?
This will help us with troubleshooting why extra launch parameters do not work for you.

Thank you!

**Comment by M1r00** [comment]
@luc1an24 thanks for your reply.
I will send you the logs for the issue asa @ home .
I updated to the last NHM 1.9.2.20 and tried to change some options. I was suspecting a soft issue as I had GPU failures on two different  rigs around 20mn of mining. 
I disabled NHM monitoring device and now the rigs are rock stable have been mining for + 24 hours. So the GPU is no longer an issue.
 I want to point out that MSI afterburner was also running ( may be too much monitoring at the same time" claymore+ NHM + afterburner").
I also enabled strap timmings through NHM however still not able to enable watchdog. May be I did an error in editing the MinerOptionsPackage.json. 
So if you know how I can enable watchdog that would be great.

M1r00 ;)
